src/reactionsystem.jl

```diff
@@ -927,7 +993,7 @@ function get_indep_sts(rs::ReactionSystem, remove_conserved = false)
     sts = get_unknowns(rs)
     nps = get_networkproperties(rs)
     indepsts = if remove_conserved
-        filter(s -> (s ∈ nps.indepspecs) && (!isbc(s)), sts)
+        filter(s -> ((s ∈ nps.indepspecs) || (!isspecies(s))) && (!isbc(s)), sts)
```

Comment:
bugfix

Reply:
I thought the point is to return the independent unknowns which include non-species as the first return argument. The independent species are returned as the second argument. There is no reason to return both anymore in this function?